src/components/Callout/style.postcss

```diff
@@ -0,0 +1,51 @@
+@import 'main.postcss';
+
+.Callout {
+  &_trigger {
+    cursor: pointer;
+    outline: 0;
+  }
+
+  &_notch,
+  &_popup {
+    position: absolute;
+  }
+
+  &_notch {
+    border-bottom: 10px solid var(--colorBorder);
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+    z-index: var(--zIndex-8--notificationsDropdownNotch);
+
+    &::after {
+      border-bottom: 9px solid #fbfbfb;
+      border-left: 9px solid transparent;
+      border-right: 9px solid transparent;
+      content: '';
+      left: -9px;
+      position: absolute;
+      top: 2px;
+    }
```

Comment:
A few variables here might be of help. I see a lot of `10px` and `9px` over the place.